packages/calcite-components/src/components/accordion/accordion.tsx

```diff
@@ -116,9 +144,17 @@ export class Accordion {
   //  Private State/Props
   //
   //--------------------------------------------------------------------------
+  mutationObserver = createObserver("mutation", () => this.updateAccordionItems());
 
   /** created list of Accordion items */
-  private items = [];
+  accordionItems: HTMLCalciteAccordionItemElement[] = [];
```

Comment:
Instead of adding `accordionItems` can we just use the existing `this.items`? It just seems like adding a redundant private property.